### ISSUE TITLE
Still getting Method not found...

### ISSUE THREAD

**Issue by vietnguyen09** [barryvdh/laravel-ide-helper#283]
Hi guys,

Im f*cking crazy whith PhpStorm when tried to fix a popular error `Method 'Bla bla' not found in class.` 

I have been searching for days in google whith hope to find out the way for this problem but no luck. 

Almost every singel topic I have read are pointing me to this laravel-ide-helper but after Install thousand times (with fresh laravel project), PhpStorm still not recognize those damn method.

I also install laravel plugin in PhpStorm but still not work too, what can I do now?

Here is my code.

```
<?php

namespace App\Http\Controllers;

use App\Article;
use App\Menu;
use Illuminate\Http\Request;
use App\Http\Requests;
use App\Http\Controllers\Controller;
use Illuminate\Http\Requestst;
use Illuminate\Support\Facades\Input;

class PagesController extends Controller
{
    public function index()
    {
        $article = Article::latest('published_at')->get(); // Method 'latest' not found in class...
        $data = array(
            'articles' => $article
        );
        return view('pages.index' , compact($data));
    }
}
```

Please help.

**Comment by mops1k** [comment]
Helper have not all methods. For example latest() not in helper list...

**Comment by EselDompteur** [comment]
Article::latest is a method of your Model I supose ... You have to make
"php artisan ide-helper:models" to fix your Problem ...

best regards

Kvintiljanov Aleksandr <notifications@github.com> schrieb am Do., 3. Dez.
2015 um 19:14 Uhr:

**Comment by vietnguyen09** [comment]
I ran "php artisan ide-helper:models" -> Yes, it wrote some line in my models file and still not work
I ran again "php artisan ide-helper:models" -> No, it created a new file called _ide_helper_models.php file but still not work.

I access file _ide_helper.php and add 

```
/**
         * Add an "order by" clause for a timestamp to the query.
         *
         * @param string $column
         * @return \Illuminate\Database\Query\Builder|static 
         * @static 
         */
        public static function latest($column = 'created_at'){
            return \Illuminate\Database\Query\Builder::latest($column);
        }
```

Still not work, even I clear cache and restart PhpStorm

**Comment by shehi** [comment]
@vietnguyen09 , make sure you don't have duplicate class declaration, as after _ide_helper_models.php is created, it copies model declarations in. You should take docblocks from that file and put them in your actual models and then remove the _ide_helper_models.php file. You might still miss some Models related methods (especially you extend ```Model``` instead of ```\Eloquent```), but we have issues discussing those particular cases in detail, please refer to them.

**Comment by yurtesen** [comment]
latest is a method in query builder
https://laravel.com/api/5.3/Illuminate/Database/Query/Builder.html#method_latest
But this works in my laravel 5.2 installation with ide-helper, I have exactly what you posted as code in my _ide_helper.php file and it seems to be enough.
In the past I had a problem with phpstorm laravel plugin when I did not open the project directory directly. If I opened a folder few folders outside, it did not work well with models. Probably not related, but just saying. There is a path setting in phpstorm laravel plugin but I couldnt get it to work. Now I only open the folder where my composer.json sits....
